ods_ci/tests/Tests/1000__model_serving/1006__model_serving_ovms_on_kserve_cli.robot

```diff
@@ -0,0 +1,109 @@
+*** Settings ***
```

Comment:
There is already a test suite for OVMS_on_kserve, why not extend that one?

Reply:
It does have UI tests in it right now since those are the only tests we have for OVMS on Kserve, I think it can be extended to have CLI tests as well